travis.sh

```diff
@@ -11,13 +12,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-language: java
-jdk:
-- oraclejdk7
-- oraclejdk8
-script: mvn verify
-after_success:
-- bash <(curl -s https://codecov.io/bash)
-branches:
-  only:
-  - master
+
+set -e
+set -x
+# Set pipefail so that `egrep` does not eat the exit code.
+set -o pipefail
+
+mvn --batch-mode clean verify | egrep -v "(^\[INFO\] Download|^\[INFO\].*skipping)"
+
```

Comment:
The test-localhost.sh script doesn't have the gcloud:run command, so there wasn't anything that I needed to get from that repo. Hopefully this branch can get deleted soon with the new plugins.